lms/djangoapps/instructor/access.py

```diff
@@ -75,6 +79,8 @@ def _change_access(course, user, level, action):
         raise ValueError("unrecognized level '{}'".format(level))
 
     if action == 'allow':
+        if level == 'ccx_coach':
+            before, after, enrollment_obj = enroll_email(course_id=course.id, student_email=user.email, auto_enroll=True)
```

Comment:
Here's the call to enroll_email. And the method signature for enroll email is:

```def enroll_email(course_id, student_email, auto_enroll=False, email_students=False, email_params=None, language=None):```

Quoting the comments on that method, ````email_students` determines if student should be notified of action by email.```

So by not specifying that parameter, the default is used, and the new coach is not emailed to notify them they've been enrolled.